### PR TITLE
feat: add cli for generating components from svgs

### PR DESCRIPTION
**Pull request by jquense**
split out some of the icon generation tooling.

### CODE REVIEW

File: packages/svg-to-component/lib.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+);
+var Svg = React.forwardRef(function (props, ref) {
+  var next = { ref: ref };
+  for (key in props) if (props.hasOwnProperty(key)) {
+    next[key] = props[key];
+  }
+  return React.cloneElement(element, next);
+});
+Svg.displayName = "${displayName}";
+SVg.element = element;
```
Comment (AaronBuxbaum) [inline packages/svg-to-component/lib.js]: accidental uppercase of `V`

File: packages/svg-to-component/test/fixtures/flower.svg [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	c-3.877,3.579-8.901,5.66-14.172,5.87l0,0c-11.564,0.462-20.831,9.73-21.293,21.293c-0.21,5.271-2.291,10.296-5.87,14.172l0,0
+	c-7.85,8.503-7.85,21.61,0,30.113c3.579,3.877,5.66,8.901,5.87,14.172c0.462,11.564,9.73,20.832,21.293,21.293l0,0
+	c5.271,0.21,10.296,2.291,14.172,5.87l0,0c8.503,7.85,21.61,7.85,30.113,0l0,0c3.877-3.579,8.901-5.66,14.172-5.87l0,0
+	c11.564-0.462,20.831-9.73,21.293-21.293c0.21-5.271,2.291-10.296,5.87-14.172C320.242,268.203,320.242,255.095,312.392,246.593z"/>
+<path style="fill:#FADC60;" d="M447.747,93.032c-5.67-0.509-10.815,1.79-14.215,5.664c-5.485,6.248-12.316,11.149-20.087,14.1
+	c-28.274,10.736-54.767,24.919-79.794,42.799c-25.518,18.231-48.694,39.65-69.46,63.122c-8.973,10.142-18.631,20.985-25.596,32.651
+	c-6.217,10.413-2.82,24.874,9.116,29.846c9.751,4.062,20.967-0.395,25.882-9.746c5.519-10.501,12.188-20.454,19.047-30.113
+	c17.929-25.25,38.699-49.042,62.651-68.748c18.763-15.437,38.604-28.964,59.803-40.257c8.885-4.734,19.016-7.02,29.007-5.778
+	c0.683,0.084,1.379,0.129,2.086,0.129c9.625,0,17.368-8.061,16.843-17.802C462.587,100.633,455.989,93.771,447.747,93.032z"/>
+<g>
```
Comment (AaronBuxbaum) [inline packages/svg-to-component/test/fixtures/flower.svg]: what's with all of these empty groups?

File: packages/svg-to-component/lib.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+// [GENERATED FILE; DO NOT EDIT BY HAND]
+
+${reactImport}
+
+var element = (
+  ${(await svgo.optimize(source)).data.trim()}
+);
+var Svg = React.forwardRef(function (props, ref) {
+  var next = { ref: ref };
+  for (key in props) if (props.hasOwnProperty(key)) {
```
Comment (AaronBuxbaum) [inline packages/svg-to-component/lib.js]: might want to put a new line or a bracket here -- even if it's not meant to be touched, still relevant IMO

Comment (taion) [reply packages/svg-to-component/lib.js]: why do we do this anyway? is the idea that this output is necessarily ES5?

Comment (jquense) [reply packages/svg-to-component/lib.js]: just avoiding it needing to be compiled

File: packages/svg-to-component/test/fixtures/flower.svg (hunk 2 above)
Comment (jquense) [reply packages/svg-to-component/test/fixtures/flower.svg]: idk i yanked it off of an icon site

**Pull request merged**